src/main/java/bio/terra/app/controller/ApiValidationExceptionHandler.java

```diff
@@ -20,6 +22,25 @@
 @ControllerAdvice
 public class ApiValidationExceptionHandler extends ResponseEntityExceptionHandler {
 
+    /**
+     * Since the default {@link ResponseEntityExceptionHandler#handleExceptionInternal}
+     * method is often passed a null body, we override it and substitute the exception's error message
+     * into an ErrorModel and return that in the response instead.
+     */
+    @Override
+    protected ResponseEntity<Object> handleExceptionInternal(
+        Exception ex, @Nullable Object body, HttpHeaders headers, HttpStatus status, WebRequest request) {
+
+        if (HttpStatus.INTERNAL_SERVER_ERROR.equals(status)) {
+            request.setAttribute(WebUtils.ERROR_EXCEPTION_ATTRIBUTE, ex, WebRequest.SCOPE_REQUEST);
+        }
+
+        Object responseBody = body == null ? new ErrorModel().message(status.name() + " - see error details")
```

Comment:
Is there a reason to prefer `name()` over `toString()` here? The `toString()` version includes both the HTTP code and the name.
```suggestion
        Object responseBody = body == null ? new ErrorModel().message(status + " - see error details")
```

Reply:
I've always used `name()` for enums. The code is already returned in the response code so I thought it was a little redundant.

Reply:
Why would you use `name()` for printing out enums? In most cases it's the same as `toString()`, and in this case `toString()` is more informative.

Reply:
TIL. Its been changed.